packages/dapp/src/components/DualWidget/DualWidget.tsx

```diff
@@ -60,6 +61,9 @@ export function DualWidget() {
         overflowX: 'hidden',
       }}
     >
+      {import.meta.env.MODE === 'testnet' && (
+        <TestnetAlert url={import.meta.env.VITE_JUMPER_URL_PROD} />
```

Comment:
initiating const in testnet alert prop would remove the need to pass the url as prop